src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -18,21 +18,49 @@ import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.gra
 
 const SHOW_TIMER_STATUSES = ["WINNING", "OUTBID", "RESERVE_NOT_MET"]
 
-interface Bid {
-  display: string
-  cents: number
+export interface BidderPositionResult {
+  status:  // bidder position status
+    | "OUTBID"
+    | "PENDING"
+    | "RESERVE_NOT_MET"
+    | "WINNING"
+    // mutation status
+    | "SALE_CLOSED"
+    | "LIVE_BIDDING_STARTED"
+    | "BIDDER_NOT_QUALIFIED"
+    // general error status for e.g. Gravity not available, no internet in the device
+    | "ERROR"
+
+  message_header: string
+  message_description_md: string
+  position: {
+    id: string
+    suggested_next_bid: {
+      cents: string
+      display: string
+    }
+  }
 }
 
 interface BidResultProps {
   sale_artwork: BidResult_sale_artwork
-  winning: boolean
-  status: string
-  message_header?: string
-  message_description_md?: string
-  suggested_next_bid?: Bid
+  bidderPositionResult: BidderPositionResult
```

Comment:
#minor but if causality ever takes over bidding the name 'bidder position' will be inaccurate. In a slightly different situation in pulse we just call this a bid result.

Reply:
`BidderPositionResult` is a name that exists in MP, so we should change MP and then Emission if that's the case.

Reply:
Yeah I think we'd probably access Causality's API through a stitched version in Metaphysics anyway, so the remapping of `BidderPositionResult` could happen there instead of within clients – GraphQL is so cool! 😄